Makefile

```diff
@@ -0,0 +1,20 @@
+build-dev:
+	cargo build --tests
```

Comment:
Looks like `build-dev` is only used by `coverage`, you can just do `RUSTFLAGS="-Cinstrument-coverage" cargo test --all-targets`. `--all-targets` includes `--tests`, also it will automatically build all the libs and bins before running all the tests.